src/mod/docs/index.md

```diff
@@ -1,8 +1,8 @@
 # Moderator Documentation
 
 ## Roles
-
-| Role Name                                                                                    | Role Requirement                                                      | Role Perks                                                                                                                                                                        |
+### Tier roles
+| Role Name                                                                                    | Level Requirement                                                      | Role Perks                                                                                                                                                                        |
```

Comment:
"Level requirement" and then each requirement being prefixed with "Solaris Level" is confusing, I prefer Role Requirement to keep consistency